test/json/test-core.json

```diff
@@ -0,0 +1,140 @@
+[
+  { "src": "hello", "exp": "hello" },
+  { "src": "hello {world}", "exp": "hello world" },
```

Comment:
comments would be useful in this file. It took me a second to remember that `world` is an unquoted.